openmp/libomptarget/include/omptarget.h

```diff
@@ -273,6 +273,7 @@ struct __tgt_target_non_contig {
 extern "C" {
 #endif
 
+void ompx_dump_mapping_tables();
```

Comment:
```suggestion
void ompx_dump_mapping_tables(void);
```
Nit. It doesn't matter here since we only use this from  C++, but since the ones below use it I suppose it keeps it consistent?